js/app/store/list.js

```diff
@@ -0,0 +1,24 @@
+import i18n from '../i18n'
+
+document.querySelectorAll('.delete-store').forEach((el) => {
```

Comment:
Let's see if we can generalize this when you implement deletion for other objects, to avoid re-implementing it multiple times.

Reply:
could be a small util function that gets selector, object name, object id, ect